plugins/oauth/components/LoginPage.js

```diff
@@ -104,23 +105,25 @@ class LoginPage extends mixin(StoreMixin) {
         </div>
         <Modal
           onClose={this.handleModalClose}
-          open={this.state.showClusterError}
+          open={true}
```

Comment:
Forgotten piece?

Reply:
Oh my - you are totally right. Good catch!!